src/vs/workbench/test/common/workbenchTestServices.ts

```diff
@@ -176,7 +176,7 @@ export class TestWorkingCopy extends Disposable implements IWorkingCopy {
 	private readonly _onDidSave = this._register(new Emitter<IStoredFileWorkingCopySaveEvent>());
 	readonly onDidSave = this._onDidSave.event;
 
-	readonly capabilities = WorkingCopyCapabilities.None;
+	readonly capabilities: WorkingCopyCapabilities = WorkingCopyCapabilities.None;
```

Comment:
Not needed, TS will infer the type from the value.

Reply:
another phantom error, yesterday it was complaining that capabilities was of _type_ `WorkingCopyCapabilities.None` so I couldn't override it. Seems fine now tho